halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/services/v1/ArtifactService.java

```diff
@@ -61,17 +62,63 @@
   @Autowired
   VersionsService versionsService;
 
+  @Autowired
+  ArtifactSourcesConfig artifactSourcesConfig;
+
   BillOfMaterials getBillOfMaterials(String deploymentName) {
     DeploymentConfiguration deploymentConfiguration = deploymentService.getDeploymentConfiguration(deploymentName);
     String version = deploymentConfiguration.getVersion();
     return versionsService.getBillOfMaterials(version);
   }
 
+  /**
+   * Should use {@link #getArtifactSources(String, SpinnakerArtifact)} when it supports all types of
```

Comment:
Which types of deployments are missing?

Reply:
Please test against the v1 k8s deployment, and localdebian, or provide detailed instructions for someone down the road to implement this.

Reply:
`BakeDebianServiceProvider` and `LocalDebianServiceProvider`